### PR TITLE
feat: Let user clear menses tags, closes #205

### LINKED ISSUE #205: Let the user delete menstruation tag(s) after deleting all entries
Deleting entries is now possible after #195, and that should make it possible to change your menstrual tag.

At a later point it should be allowed to change at any point: #208

### PR DESCRIPTION
**Pull request by olavea**

### CODE REVIEW

File: src/features/profile/DangerCard.js [hunk 1]
```diff
@@ -95,6 +96,17 @@ const DangerCard = () => {
...
           {deleteAllEntriesText}
         </Button>
+
+        <Button
+          // true if entrie.length > 0
+          disabled={disabled}
+          // alert "Clearing tags" when clicked. (This will later be replaced with proper functionality).
+          variant="outlined"
+          color="primary"
+          // onClick={clearMensenTags}
```
Comment (raae) [inline src/features/profile/DangerCard.js:106]: Create a `handleClearMensesTags` where you do all the setPending stuff and error handling. Take a look at `handleDeleteAllEntries`. (No need to have "confirm" alert here as in delete all entries).

File: src/features/profile/DangerCard.js [hunk 2]
```diff
@@ -65,8 +66,18 @@ const DangerCard = () => {
 
     setIsPending(false)
   }
+  const disabled2 = entries.length > 0
```
Comment (raae) [inline src/features/profile/DangerCard.js:69]: **suggestion:** Also disable when pending, you need to use and or for that `||`
**suggestion:** Rename variable `disabledMensenTags` or something like that more descriptive.

**Review by raae — COMMENTED** [review]
**praise:** Great job so far, liking this a lot 🎉

**Review by raae — COMMENTED** [review]
🎉

**Comment by olavea** [comment]
Why is the button grey?
When I change it to === 0 it works....

**Comment by raae** [comment]
It should be gray when there are entries and not gray when are no entries. So after clicking delete all entries it should turn non grey.

It might be that we forget to set pending to false after delete all is done?

**Comment by olavea** [comment]
Is this the correct way to call the function?
`const { error } = await dispatch(fakeFunctionClearMensesTags())`

What more do I need to do here? 

- [ ] error stuff?
- [ ] pending stuff?

**Comment by raae** [comment]
I can take it from here and you will be the reviewer then :D

**Comment by raae** [comment]
I am not allowed to ask you to review as its your PR, but take it for a spin and let me know if it works like it should.

**Review by raae — APPROVED** [review]

**Pull request merged**